dev/com.ibm.ws.org.apache.cxf.cxf.rt.frontend.jaxrs.3.2/src/org/apache/cxf/jaxrs/provider/ProviderFactory.java

```diff
@@ -1704,8 +1812,10 @@ protected static boolean filterContractSupported(ProviderInfo<?> provider,
 
     public void setProviderComparator(Comparator<?> providerComparator) {
         this.providerComparator = providerComparator;
-        sortReaders();
-        sortWriters();
+        //Liberty code change start
+        addAndSortReaders(new ArrayList<>(0));
+        addAndSortWriters(new ArrayList<>(0));
```

Comment:
same comment as before - can we pass an empty list constant?  or just sort the existing reader/writer providers without adding an empty list?